test/named_columns.test.js

```diff
@@ -29,7 +29,7 @@ describe('named columns', function() {
 
     it('should be able to retrieve rowid of last inserted value', function(done) {
         db.get("SELECT last_insert_rowid() as last_id FROM foo", function(err, row) {
-            if (err) throw err;
+            if (err) done(err);
```

Comment:
Haven't look into other test cases, but errors should be passed to `done` callback.